stack/syncGateway.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+echo "Running syncGateway.sh"
+
+version=$1
+
+echo "Sleeping to prevent dpkg lock"
+sleep 20s #workaround for dpkg lock issue
+
+echo "Using the settings:"
+echo version \'$version\'
+
+echo "Installing Couchbase Sync Gateway..."
+wget https://packages.couchbase.com/releases/couchbase-sync-gateway/${version}/couchbase-sync-gateway-enterprise_${version}_x86_64.deb
+dpkg -i couchbase-sync-gateway-enterprise_${version}_x86_64.deb
+
+echo "Calling util.sh..."
+source util.sh
+adjustTCPKeepalive
+
+echo "Configuring Couchbase Sync Gateway..."
+file="/home/sync_gateway/sync_gateway.json"
+echo '
```

Comment:
One day - if you plan on creating a bucket during couchbase init... would be cool to replicate sync gateway to it during install here